diff --git a/debian/control b/debian/control
index 569cc45..df585a2 100644
--- a/debian/control
+++ b/debian/control
@@ -3,7 +3,6 @@ Maintainer: Julien Lamy <lamy@unistra.fr>
 Section: science
 Priority: optional
 Build-Depends: debhelper (>= 9), 
-               dh-python,
                cmake,
                pkg-config,
                libqt4-dev,
diff --git a/debian/rules b/debian/rules
index ffa8b18..f3ab514 100755
--- a/debian/rules
+++ b/debian/rules
@@ -1,7 +1,6 @@
 #!/usr/bin/make -f
 
 export DEB_CXXFLAGS_MAINT_APPEND  = -std=c++0x
-export DEB_BUILD_MAINT_OPTIONS = hardening=+all
 
 %:
    dh $@ --builddirectory=build --parallel --with python2
