diff --git a/debian/control b/debian/control
index 326b841..683e856 100644
--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,7 @@ Source: dicomifier
 Maintainer: Julien Lamy <lamy@unistra.fr>
 Section: science
 Priority: optional
-Build-Depends: debhelper (>= 10), 
+Build-Depends: debhelper (>= 9), 
                d-shlibs,
                dh-python,
                cmake,
diff --git a/debian/rules b/debian/rules
index e3af64f..e109caf 100755
--- a/debian/rules
+++ b/debian/rules
@@ -71,6 +71,11 @@ override_dh_install-arch:
 		--override s/libdcm.*-dev/libdcmtk-dev/ \
 		--override s/libof.*-dev/libdcmtk-dev/ \
 		--override s/libjsoncpp.-dev/libjsoncpp-dev/ \
+		--override s/libicuuc.*-dev/libicu-dev/ \
+		--override s/libboost_date_time.*-dev/libboost-date-time-dev/ \
+		--override s/libboost_filesystem.*-dev/libboost-filesystem-dev/ \
+		--override s/libboost_regex.*-dev/libboost-regex-dev/ \
+		--override s/libboost_system.*-dev/libboost-system-dev/ \
 		debian/tmp/usr/lib/*.so
 
 override_dh_clean:
