diff --git a/debian/rules b/debian/rules
index ae7efce..b2a02e2 100755
--- a/debian/rules
+++ b/debian/rules
@@ -54,7 +54,7 @@ override_dh_auto_build-indep:
 	find documentation/_build -name "*.md5" -delete
 
 override_dh_makeshlibs-arch:
-	dh_makeshlibs -V'libodil0 (>= 0.9.2-1)'
+	dh_makeshlibs -V'libodil0 (>= 0.9.2-1~bionic)'
 
 # Run C++ tests only on base build
 override_dh_auto_test-arch-nopy:
 
