diff --git a/debian/compat b/debian/compat
index f599e28..ec63514 100644
--- a/debian/compat
+++ b/debian/compat
@@ -1 +1 @@
-10
+9
diff --git a/debian/control b/debian/control
index e18d84f..a43b2fd 100644
--- a/debian/control
+++ b/debian/control
@@ -3,14 +3,14 @@ Maintainer: Debian Med Packaging Team <debian-med-packaging@lists.alioth.debian.
 Uploaders: Julien Lamy <lamy@unistra.fr>
 Section: science
 Priority: optional
-Build-Depends: debhelper (>= 10),
-               d-shlibs (>= 0.81),
+Build-Depends: debhelper (>= 9),
+               d-shlibs,
                dh-python,
                cmake,
-               g++ (>= 4:7),
+               g++ (>= 4:4.9),
                pkg-config,
                libjsoncpp-dev,
-               libdcmtk-dev | libdcmtk2-dev,
+               libdcmtk-dev,
                libicu-dev (>= 4.8.1.1),
                zlib1g-dev,
                uuid-dev,
diff --git a/debian/rules b/debian/rules
index 65756ce..5094e63 100755
--- a/debian/rules
+++ b/debian/rules
@@ -1,6 +1,6 @@
 #!/usr/bin/make -f
 
-export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all,-pie
 
 # Find all Python versions
 PYTHON2=$(shell pyversions -vrd)
@@ -8,7 +8,7 @@ PYTHON3=$(shell py3versions -vrd)
 ALLPY=$(PYTHON2) $(PYTHON3)
 
 %:
-	dh $@ --builddirectory=build --with python2,python3
+	dh $@ --builddirectory=build --with python2,python3 --parallel
 
 # Base build: no wrappers
 override_dh_auto_configure-arch:
@@ -99,6 +99,14 @@ override_dh_install-arch:
 		--commit --multiarch --exclude-a --exclude-la \
 		--movedev debian/tmp/usr/include/* usr/include \
 		--override s/libboost_log.*-dev/libboost-log-dev/ \
+		--override s/libjsoncpp.-dev/libjsoncpp-dev/ \
+		--override s/libdcm.*-dev/libdcmtk-dev/ \
+		--override s/libof.*-dev/libdcmtk-dev/ \
+		--override s/libicuuc.*-dev/libicu-dev/ \
+		--override s/libboost_date_time.*-dev/libboost-date-time-dev/ \
+		--override s/libboost_filesystem.*-dev/libboost-filesystem-dev/ \
+		--override s/libboost_regex.*-dev/libboost-regex-dev/ \
+		--override s/libboost_system.*-dev/libboost-system-dev/ \
 		debian/tmp/usr/lib/*.so
 
 override_dh_clean:
diff --git a/wrappers/python/CMakeLists.txt b/wrappers/python/CMakeLists.txt
index c0bb792..edeb38a 100644
--- a/wrappers/python/CMakeLists.txt
+++ b/wrappers/python/CMakeLists.txt
@@ -20,7 +20,8 @@ list(SORT Python_Files)
 
 pybind11_add_module(pyodil SHARED ${Source_Files} ${Header_Files})
 
-set_target_properties(pyodil PROPERTIES OUTPUT_NAME _odil FOLDER "Wrappers")
+set_target_properties(
+    pyodil PROPERTIES OUTPUT_NAME _odil FOLDER "Wrappers" NO_SONAME ON)
 if(APPLE)
     set_target_properties(pyodil PROPERTIES SUFFIX ".so")
 endif()
