diff --git a/debian/control b/debian/control
index 3c86b04..f041e49 100644
--- a/debian/control
+++ b/debian/control
@@ -3,14 +3,14 @@ Maintainer: Debian Med Packaging Team <debian-med-packaging@lists.alioth.debian.
 Uploaders: Julien Lamy <lamy@unistra.fr>
 Section: science
 Priority: optional
-Build-Depends: debhelper (>= 10),
-               d-shlibs (>= 0.81),
+Build-Depends: debhelper (>= 9),
+               d-shlibs,
                dh-python,
                cmake,
-               g++ (>= 4:7),
+               g++ (>= 4:4.9),
                pkg-config,
                libjsoncpp-dev,
-               libdcmtk-dev | libdcmtk2-dev,
+               libdcmtk-dev,
                libicu-dev (>= 4.8.1.1),
                zlib1g-dev,
                uuid-dev,
diff --git a/debian/rules b/debian/rules
index ae7efce..723de3a 100755
--- a/debian/rules
+++ b/debian/rules
@@ -109,6 +109,14 @@ override_dh_install-arch:
 		--devunversioned \
 		--commit --multiarch --exclude-a --exclude-la \
 		--movedev debian/tmp/usr/include/* usr/include \
+		--override s/libjsoncpp.-dev/libjsoncpp-dev/ \
+		--override s/libdcm.*-dev/libdcmtk-dev/ \
+		--override s/libof.*-dev/libdcmtk-dev/ \
+		--override s/libicuuc.*-dev/libicu-dev/ \
+		--override s/libboost_date_time.*-dev/libboost-date-time-dev/ \
+		--override s/libboost_filesystem.*-dev/libboost-filesystem-dev/ \
+		--override s/libboost_regex.*-dev/libboost-regex-dev/ \
+		--override s/libboost_system.*-dev/libboost-system-dev/ \
 		debian/tmp/usr/lib/*.so
 
 
