diff -u a/debian/control b/control
--- a/debian/control	2016-04-21 19:43:08.463018000 +0200
+++ b/control	2016-04-23 15:18:28.494920968 +0200
@@ -5,11 +5,11 @@
 Priority: optional
 Build-Depends: debhelper (>= 9),
                d-shlibs,
-               dh-python,
                cmake,
                pkg-config,
                libjsoncpp-dev,
-               libdcmtk-dev | libdcmtk2-dev,
+               libdcmtk2-dev,
+               libwrap0-dev,
                libicu-dev (>= 4.8.1.1),
                zlib1g-dev,
                uuid-dev,
@@ -33,6 +33,7 @@ Architecture: any
 Section: libs
 Depends: ${shlibs:Depends},
          ${misc:Depends}
+Pre-Depends: multiarch-support
 Provides: libdcmtkpp0
 Conflicts: libdcmtkpp0
 Replaces: libdcmtkpp0
diff -u a/debian/rules b/rules
--- a/debian/rules	2016-04-22 19:28:14.275011000 +0200
+++ b/rules	2016-04-23 15:27:36.886920968 +0200
@@ -1,7 +1,6 @@
 #!/usr/bin/make -f
 
 export DEB_CXXFLAGS_MAINT_APPEND  = -std=c++0x
-export DEB_BUILD_MAINT_OPTIONS = hardening=+all
 
 %:
 	dh $@ --builddirectory=build --parallel --with python2
@@ -16,17 +15,19 @@
 override_dh_install-arch:
 	dh_install
 	chrpath -d debian/python-odil/usr/lib/python*/dist-packages/odil.so
+	touch debian/tmp/usr/lib/libodil.a
 	d-shlibmove \
-		--commit --multiarch --exclude-a --exclude-la \
+		--commit --multiarch --exclude-la \
 		--movedev debian/tmp/usr/include/* usr/include \
 		--override s/libjsoncpp.-dev/libjsoncpp-dev/ \
-		--override s/libdcm.*-dev/libdcmtk-dev/ \
-		--override s/libof.*-dev/libdcmtk-dev/ \
+		--override s/libdcm.*-dev/libdcmtk2-dev/ \
+		--override s/libof.*-dev/libdcmtk2-dev/ \
 		--override s/libicuuc.*-dev/libicu-dev/ \
                 --override s/libboost_filesystem.*-dev/libboost-filesystem-dev/ \
                 --override s/libboost_regex.*-dev/libboost-regex-dev/ \
                 --override s/libboost_system.*-dev/libboost-system-dev/ \
 		debian/tmp/usr/lib/*.so
+	find debian -name libodil.a -delete
 
 # These steps are not needed for arch-independent packages
 override_dh_auto_configure-indep:
