diff -ur a/debian/compat b/debian/compat
--- a/debian/compat
+++ b/debian/compat
@@ -1 +1 @@
-11
+10
diff -ur a/debian/control b/debian/control
--- a/debian/control
+++ b/debian/control
@@ -4,7 +4,7 @@
 Section: libs
 Priority: optional
 Build-Depends: cmake (>= 2.8.12),
-               debhelper (>= 11),
+               debhelper (>= 10),
                dpkg-dev (>= 1.17.14),
                dh-python,
                libboost-dev (>= 1.56) <!nocheck>,
