diff -ruN a/debian/compat b/debian/compat
--- a/debian/compat	2018-09-22 02:26:31.000000000 +0000
+++ b/debian/compat	2018-10-09 15:14:10.590000000 +0000
@@ -1 +1 @@
-11
+9
diff -ruN a/debian/control b/debian/control
--- a/debian/control	2018-09-22 02:26:31.000000000 +0000
+++ b/debian/control	2018-10-09 15:17:06.120000000 +0000
@@ -4,22 +4,22 @@
 Section: libs
 Priority: optional
 Build-Depends: cmake (>= 2.8.12),
-               debhelper (>= 11),
-               dpkg-dev (>= 1.17.14),
+               debhelper (>= 9),
+               dpkg-dev (>= 1.17.5),
                dh-python,
-               libboost-dev (>= 1.56) <!nocheck>,
-               libeigen3-dev <!nocheck>,
+               libboost-dev (>= 1.54),
+               libeigen3-dev,
                python-all,
+               python-dev,
+               python-pytest,
                python-setuptools,
                python3-all,
-               python3-breathe <!nodoc>,
-               python3-dev <!nocheck>,
-               python3-numpy <!nocheck>,
-               python3-pytest <!nocheck>,
-               python3-scipy <!nocheck>,
+               python3-dev,
+               python3-numpy,
+               python3-pytest,
+               python3-scipy,
                python3-setuptools,
-               python3-sphinx <!nodoc>,
-               python3-sphinx-rtd-theme <!nodoc>,
+               python3-sphinx,
                sphinx-common
 Standards-Version: 4.2.1
 Vcs-Browser: https://salsa.debian.org/science-team/pybind11
diff -ruN a/debian/patches/0001_Remove_breathe.patch b/debian/patches/0001_Remove_breathe.patch
--- a/debian/patches/0001_Remove_breathe.patch	1970-01-01 00:00:00.000000000 +0000
+++ b/debian/patches/0001_Remove_breathe.patch	2018-10-09 15:14:10.600000000 +0000
@@ -0,0 +1,13 @@
+Index: build/docs/conf.py
+===================================================================
+--- build.orig/docs/conf.py
++++ build/docs/conf.py
+@@ -31,7 +31,7 @@ import subprocess
+ # Add any Sphinx extension module names here, as strings. They can be
+ # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+ # ones.
+-extensions = ['breathe']
++extensions = []
+ 
+ breathe_projects = {'pybind11': '.build/doxygenxml/'}
+ breathe_default_project = 'pybind11'
diff -ruN a/debian/patches/0002_Sphinx-theme.patch b/debian/patches/0002_Sphinx-theme.patch
--- a/debian/patches/0002_Sphinx-theme.patch	1970-01-01 00:00:00.000000000 +0000
+++ b/debian/patches/0002_Sphinx-theme.patch	2018-10-09 15:14:10.600000000 +0000
@@ -0,0 +1,15 @@
+Index: build/docs/conf.py
+===================================================================
+--- build.orig/docs/conf.py	2018-10-09 15:03:00.000000000 +0000
++++ build/docs/conf.py	2018-10-09 15:04:37.240000000 +0000
+@@ -118,10 +118,6 @@
+ on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
+ 
+ if not on_rtd:  # only import and set the theme if we're building docs locally
+-    import sphinx_rtd_theme
+-    html_theme = 'sphinx_rtd_theme'
+-    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+-
+     html_context = {
+         'css_files': [
+             '_static/theme_overrides.css'
diff -ruN a/debian/patches/series b/debian/patches/series
--- a/debian/patches/series	1970-01-01 00:00:00.000000000 +0000
+++ b/debian/patches/series	2018-10-09 15:14:10.600000000 +0000
@@ -0,0 +1,2 @@
+0001_Remove_breathe.patch
+0002_Sphinx-theme.patch
diff -ruN a/debian/rules b/debian/rules
--- a/debian/rules	2018-09-22 02:26:31.000000000 +0000
+++ b/debian/rules	2018-10-09 15:14:10.600000000 +0000
@@ -10,7 +10,7 @@
 
 CONFIGURE_ARGS=-DPYBIND11_INSTALL:BOOL=ON
 ifeq (,$(filter nocheck,$(DEB_BUILD_OPTIONS)))
-CONFIGURE_ARGS+=-DPYBIND11_TEST:BOOL=ON \
+CONFIGURE_ARGS+=-DPYBIND11_TEST:BOOL=OFF \
 	-DPYTHON_EXECUTABLE:FILEPATH=/usr/bin/$(shell py3versions -d)
 else
 CONFIGURE_ARGS+=-DPYBIND11_TEST:BOOL=OFF
diff -ruN a/debian/watch b/debian/watch
--- a/debian/watch	2018-09-22 02:26:31.000000000 +0000
+++ b/debian/watch	2018-10-09 15:14:10.600000000 +0000
@@ -1,4 +1,3 @@
-version=4
-opts=uversionmangle=s/(\d)[_\.\-\+]?((RC|rc|pre|dev|beta|alpha)\d*)$/$1~$2/,\
-  filenamemangle=s/.+\/v?@ANY_VERSION@\.tar\.gz/pybind11-$1\.tar\.gz/ \
-  https://github.com/pybind/pybind11/tags .*/v?@ANY_VERSION@\.tar\.gz
+version=3
+opts=filenamemangle=s/.+\/v?(\d\S*)\.tar\.gz/pybind11-$1\.tar\.gz/ \
+  https://github.com/pybind/pybind11/tags .*/v?(\d\S*)\.tar\.gz
